### ISSUE TITLE
networking breaks functional tests when run as non-priv user

### ISSUE THREAD

**Issue by jodh-intel** [01org/cc-oci-runtime#93]
`make functional-test` should run as a non-root user, but if you try that now, you get this error:

```
 2016-08-08T17:42:05.062873Z:15654:cc-oci-runtime:critical:failed to run prestart hooks
   
   2016-08-08T17:42:05.064133Z:15654:cc-oci-runtime:critical:Failed to create tap [cwlp3s0] [Operation not permitted]
   
   2016-08-08T17:42:05.064164Z:15654:cc-oci-runtime:critical:failed to create network
   
   2016-08-08T17:42:05.064193Z:15654:cc-oci-runtime:critical:failed to launch VM

```

The runtime should detect if it is being run as a non-root user and disable network setup in that scenario along with a warning message that networking is disabled.

**Comment by jodh-intel** [comment]
We should run all tests in Travis as a non-root user and then again as `root` (see #69).

**Comment by jodh-intel** [comment]
Fixed by #216.
